Fixes #11764 - Location's parameters can be accessed through API

xprazak2 opened the pull request:
We no longer get an error when calling /api/v2/locations/:id/parameters for a location with parameters

Commit:
Fixes #11764 - Taxonomy parameters can be accessed through API

dLobatog commented:
Thanks @xprazak2 ! This would be the case for Organizations as well, can you make sure organizations/:id/parameters works too? A test would be necessary to make sure we don't get any regressions here.

Thanks again!

dLobatog commented:
Forgot to mention, I updated the Redmine ticket to be more generic so please update the commit.  :smile:

xprazak2 commented:
@eLobato, I added tests, changed commit message and handled the case of Organizations, but their route gets overridden by [Katello](https://github.com/Katello/katello/blob/master/config/routes/overrides.rb#L2), so tests are green but in reality we get 404. I would expect at least a message that the route is forbidden for whatever reason.

dLobatog commented:
@xprazak2 I think that's fine, we're only concerned about Foreman in this repo - you should submit a patch to Katello though to fix this behavior there.

dLobatog commented:
Merged as e877b62fc77efdc706bdbd0302fae46ae420c490, thanks @xprazak2!
